src/Surfnet/ServiceProviderDashboard/Application/Assembler/ServiceStatusAssembler.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+//declare(strict_types = 1);
+
```

Comment:
Either enable, or remove if strict types are not relevant here.

Reply:
removed